src/main/twirl/views/sectionPreview.scala.html

```diff
@@ -27,10 +29,28 @@ <h1 class="heading-xlarge">
 
             @fields.map(_.renderPreview(answers))
 
-            <div class="rifs-form-buttons">
-                <a class="button secondary" href="@controllers.routes.ApplicationController.show(app.id)">Close</a>
-                <a class="flat" href="@controllers.routes.ApplicationController.showSectionForm(app.id, formSection.sectionNumber)">Edit this page</a>
-            </div>
+            @if(editButton.isDefined){
+                <p></p>
```

Comment:
yes both are removed now (I was forgetting that the page was already productionised which means my fiddling is probably a little irritating!)